### PR TITLE
Fix WPCS: PSR2 and PEAR sniffs

### PR DESCRIPTION
**Pull request by pedro-mendonca**
See #585 

Fixes:
- [x] `PEAR.Methods.MethodDeclaration.StaticBeforeVisibility`
- [x] `PEAR.Methods.MethodDeclaration.AbstractAfterVisibility`
- [x] `PEAR.ControlStructures.SwitchDeclaration.SpaceBeforeColonCASE`
- [x] `PEAR.ControlStructures.ElseIfDeclaration.NotAllowed`
- [x] `PEAR.Methods.MethodDeclaration.StaticBeforeVisibility`
- [x] `PEAR.Functions.FunctionCallSignature.SpaceBeforeOpenBracket`
- [x] `PEAR.Functions.FunctionCallSignature.ContentAfterOpenBracket`
- [x] `PEAR.Functions.FunctionCallSignature.SpaceAfterCloseBracket`
- [x] `PEAR.Functions.FunctionCallSignature.SpaceAfterOpenBracket`
- [x] `PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket`
- [x] `PEAR.Functions.FunctionCallSignature.SpaceBeforeOpenBracket`
- [x] `PEAR.Functions.FunctionCallSignature.MultipleArguments`
- [x] `PEAR.Functions.FunctionCallSignature.SpaceBeforeCloseBracket`
- [x] `PEAR.Files.IncludingFile.UseRequireOnce`
- [x] `PEAR.Files.IncludingFile.BracketsNotRequired`

### CODE REVIEW

**Comment by pedro-mendonca** [comment]
I will not add any commits to this PR.
I'll push the next one with `WordPress` and `Squiz` sniffs after this one is merged.

File: gp-includes/cli/remove-multiple-currents.php [hunk 1]
```diff
@@ -6,19 +6,24 @@ public function __invoke() {
...
+				),
+				'original_id ASC'
+			);
 			$prev_original_id = null;
 			foreach ( $translations as $translation ) {
 				if ( $translation->original_id == $prev_original_id ) {
-					WP_CLI::warning( sprintf(
+					WP_CLI::warning(
+						sprintf(
 						/* translators: 1: original ID, 2: new ID */
```
Comment (ocean90) [inline gp-includes/cli/remove-multiple-currents.php]: The comment isn't aligned/indented.

File: gp-includes/routes/profile.php [hunk 2]
```diff
@@ -64,15 +64,18 @@ public function profile_view( $user = '' ) {
 	private function get_recent_translation_sets( $user, $amount = 5 ) {
 		global $wpdb;
 
-		$translations = GP::$translation_set->many_no_map("
+		$translations = GP::$translation_set->many_no_map(
+			"
```
Comment (ocean90) [inline gp-includes/routes/profile.php]: Let's keep the quotes on the same line as where the SQL string starts/ends.

```
GP::$translation_set->many_no_map(
   "SELECT …
   ORDER BY date_added DESC"
   …
);
```

File: gp-includes/routes/profile.php [hunk 3]
```diff
@@ -122,14 +125,17 @@ private function get_recent_translation_sets( $user, $amount = 5 ) {
 	private function locales_known( $user ) {
 		global $wpdb;
 
-		$translations = GP::$translation_set->many_no_map("
+		$translations = GP::$translation_set->many_no_map(
+			"
```
Comment (ocean90) [inline gp-includes/routes/profile.php]: Keep quotes on the same line, see above.

File: gp-includes/things/glossary.php [hunk 4]
```diff
@@ -151,13 +154,17 @@ public function copy_glossary_items_from( $source_glossary_id ) {
 
 		$current_date = $this->now_in_mysql_format();
 
-		return $this->query("
+		return $this->query(
+			"
```
Comment (ocean90) [inline gp-includes/things/glossary.php]: Keep quotes on the same line, see above.

File: gp-includes/things/original.php [hunk 5]
```diff
@@ -120,17 +120,21 @@ public function count_by_project_id( $project_id, $type = 'total' ) {
...
-				COUNT( CASE WHEN priority <> '-2' THEN priority END ) AS `public`
-			FROM {$wpdb->gp_originals}
-			WHERE
-				project_id = %d AND status = '+active'
-			",
-			$project_id
-		), ARRAY_A );
+		$counts = $wpdb->get_row(
+			$wpdb->prepare(
+				"
```
Comment (ocean90) [inline gp-includes/things/original.php]: Keep quotes on the same line, see above.

File: gp-includes/things/project.php [hunk 6]
```diff
@@ -385,13 +387,16 @@ public function copy_sets_and_translations_from( $source_project_id ) {
 
 	public function copy_originals_from( $source_project_id ) {
 		global $wpdb;
-		return $this->query("
+		return $this->query(
+			"
```
Comment (ocean90) [inline gp-includes/things/project.php]: Keep quotes on the same line, see above.

File: gp-includes/things/translation-set.php [hunk 7]
```diff
@@ -171,26 +171,36 @@ public function name_with_locale( $separator = '&rarr;' ) {
 	}
 
 	public function by_project_id_slug_and_locale( $project_id, $slug, $locale_slug ) {
-		$result = $this->one( "
+		$result = $this->one(
+			"
```
Comment (ocean90) [inline gp-includes/things/translation-set.php]: Keep quotes on the same line, see above.

File: gp-includes/things/translation-set.php [hunk 8]
```diff
@@ -171,26 +171,36 @@ public function name_with_locale( $separator = '&rarr;' ) {
...
+			);
 		}
 
 		return $result;
 	}
 
 	public function by_locale( $locale_slug ) {
-		return $this->many( "
+		return $this->many(
+			"
```
Comment (ocean90) [inline gp-includes/things/translation-set.php]: Keep quotes on the same line, see above.

File: gp-includes/things/translation-set.php [hunk 9]
```diff
@@ -171,26 +171,36 @@ public function name_with_locale( $separator = '&rarr;' ) {
...
 		    SELECT * FROM $this->table
-		    WHERE locale = %s", $locale_slug );
+		    WHERE locale = %s",
+			$locale_slug
```
Comment (ocean90) [inline gp-includes/things/translation-set.php:200]: Incorrect indentation.

File: gp-includes/things/translation-set.php [hunk 10]
```diff
@@ -206,9 +216,12 @@ public function existing_slugs() {
 	}
 
 	public function by_project_id( $project_id ) {
-		return $this->many( "
+		return $this->many(
+			"
```
Comment (ocean90) [inline gp-includes/things/translation-set.php]: Keep quotes on the same line, see above.

File: gp-includes/things/translation-set.php [hunk 11]
```diff
@@ -206,9 +216,12 @@ public function existing_slugs() {
...
 		    SELECT * FROM $this->table
-		    WHERE project_id = %d ORDER BY name ASC", $project_id );
+		    WHERE project_id = %d ORDER BY name ASC",
+			$project_id
```
Comment (ocean90) [inline gp-includes/things/translation-set.php:220]: Incorrect indentation.

File: gp-includes/things/translation-set.php [hunk 12]
```diff
@@ -474,38 +497,48 @@ public function update_status_breakdown() {
...
+						COUNT(*) AS total,
+						COUNT( CASE WHEN o.priority = '-2' THEN o.priority END ) AS `hidden`,
+						COUNT( CASE WHEN o.priority <> '-2' THEN o.priority END ) AS `public`
+					FROM {$wpdb->gp_translations} AS t
+					INNER JOIN {$wpdb->gp_originals} AS o ON t.original_id = o.id
+					WHERE
+						t.translation_set_id = %d
+						AND o.status = '+active'
+					GROUP BY t.status
+					",
```
Comment (ocean90) [inline gp-includes/things/translation-set.php]: Keep quotes on the same line, see above.

File: gp-includes/things/translation-set.php [hunk 13]
```diff
@@ -474,38 +497,48 @@ public function update_status_breakdown() {
...
-				WHERE
-					t.translation_set_id = %d AND
-					o.status = '+active' AND
-					( t.status = 'current' OR t.status = 'waiting' )
-					AND warnings IS NOT NULL
-					AND warnings != ''
-			", $this->id ) );
+			$warnings_counts = $wpdb->get_row(
+				$wpdb->prepare(
+					"
```
Comment (ocean90) [inline gp-includes/things/translation-set.php]: Keep quotes on the same line, see above.

File: gp-includes/things/translation-set.php [hunk 14]
```diff
@@ -516,18 +549,24 @@ public function update_status_breakdown() {
...
-				FROM {$wpdb->gp_originals} AS o
-				LEFT JOIN {$wpdb->gp_translations} AS t ON o.id = t.original_id AND t.translation_set_id = %d AND t.status != 'rejected' AND t.status != 'old'
-				WHERE
-					o.project_id = %d AND
-					o.status = '+active' AND
-					t.translation_0 IS NULL
-			", $this->id, $this->project_id ) );
+			$untranslated_counts = $wpdb->get_row(
+				$wpdb->prepare(
+					"
```
Comment (ocean90) [inline gp-includes/things/translation-set.php]: Keep quotes on the same line, see above.

File: gp-includes/things/translation-set.php [hunk 15]
```diff
@@ -594,13 +633,17 @@ public function copy_translations_from( $source_translation_set_id ) {
 				}
 			}
 		} else {
-			return $this->query( "
+			return $this->query(
+				"
```
Comment (ocean90) [inline gp-includes/things/translation-set.php]: Keep quotes on the same line, see above.

File: gp-templates/helper-functions.php [hunk 16]
```diff
@@ -17,11 +17,15 @@
...
 	// Glossaries are injected into the translations prior to escaping and prepare_original() being run.
 	$glossary_entries = array();
-	$text             = preg_replace_callback( '!(<span class="glossary-word"[^>]+>)!i', function( $m ) use ( &$glossary_entries ) {
+	$text             = preg_replace_callback(
+		'!(<span class="glossary-word"[^>]+>)!i',
+		function( $m ) use ( &$glossary_entries ) {
 		$item_number                      = count( $glossary_entries );
 		$glossary_entries[ $item_number ] = $m[0];
 		return "<span GLOSSARY={$item_number}>";
-	}, $text );
```
Comment (ocean90) [inline gp-templates/helper-functions.php]: Incorrect indentation, needs one more tab.

File: gp-templates/helper-functions.php [hunk 17]
```diff
@@ -31,9 +35,13 @@ function prepare_original( $text ) {
 	$text = preg_replace( '/(%(\d+\$(?:\d+)?)?[bcdefgosuxEFGX])/', '<span class="notranslate">\\1</span>', $text );
 
 	// Put the glossaries back!
-	$text = preg_replace_callback( '!(<span GLOSSARY=(\d+)>)!', function( $m ) use ( $glossary_entries ) {
+	$text = preg_replace_callback(
+		'!(<span GLOSSARY=(\d+)>)!',
+		function( $m ) use ( $glossary_entries ) {
 		return $glossary_entries[ $m[2] ];
-	}, $text );
+		},
```
Comment (ocean90) [inline gp-templates/helper-functions.php]: Incorrect indentation, needs one more tab.

File: gp-templates/helper-functions.php [hunk 18]
```diff
@@ -101,9 +109,12 @@ function gp_sort_glossary_entries_terms( $glossary_entries ) {
 		$glossary_entries_terms[ $key ] = implode( '|', $terms );
 	}
 
-	uasort( $glossary_entries_terms, function( $a, $b ) {
-		return gp_strlen($a) < gp_strlen($b);
-	} );
+	uasort(
+		$glossary_entries_terms,
+		function( $a, $b ) {
+		return gp_strlen( $a ) < gp_strlen( $b );
```
Comment (ocean90) [inline gp-templates/helper-functions.php]: Incorrect indentation, needs one more tab.

File: gp-templates/project.php [hunk 19]
```diff
@@ -12,11 +12,14 @@
...
 if ( $project->active ) {
-	add_filter( 'gp_breadcrumb_items', function( $items ) {
-		$items[ count($items) - 1 ] .= ' <span class="active bubble">' . __( 'Active', 'glotpress' ) . '</span>';
+	add_filter(
+		'gp_breadcrumb_items',
+		function( $items ) {
+		$items[ count( $items ) - 1 ] .= ' <span class="active bubble">' . __( 'Active', 'glotpress' ) . '</span>';
 
 		return $items;
-	} );
```
Comment (ocean90) [inline gp-templates/project.php]: Incorrect indentation, needs one more tab.

File: gp-templates/settings.php [hunk 20]
```diff
@@ -29,7 +29,7 @@
 ?>
 <h2><?php _e( 'Your Settings', 'glotpress' ); ?></h2>
 <form action="" method="post">
-	<?php include_once __DIR__ . '/settings-edit.php'; ?>
+	<?php require_once __DIR__ . '/settings-edit.php'; ?>
```
Comment (ocean90) [inline gp-templates/settings.php:32]: What was the reason for this change?

**Review by ocean90 — CHANGES_REQUESTED** [review]

File: gp-templates/settings.php (hunk 20 above)
Comment (pedro-mendonca) [reply gp-templates/settings.php:32]: This was changed by this rule:
`PEAR.Files.IncludingFile.UseRequireOnce`

File: gp-includes/cli/remove-multiple-currents.php (hunk 1 above)
Comment (pedro-mendonca) [reply gp-includes/cli/remove-multiple-currents.php]: Fixed

File: gp-includes/routes/profile.php (hunk 3 above)
Comment (pedro-mendonca) [reply gp-includes/routes/profile.php]: Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1084/commits/a68e1405be04f26ab28114ef0585e2cac832d3d4

File: gp-includes/routes/profile.php (hunk 2 above)
Comment (pedro-mendonca) [reply gp-includes/routes/profile.php]: Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1084/commits/a68e1405be04f26ab28114ef0585e2cac832d3d4

File: gp-includes/things/glossary.php (hunk 4 above)
Comment (pedro-mendonca) [reply gp-includes/things/glossary.php]: Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1084/commits/a68e1405be04f26ab28114ef0585e2cac832d3d4

File: gp-templates/helper-functions.php (hunk 16 above)
Comment (pedro-mendonca) [reply gp-templates/helper-functions.php]: Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1084/commits/a68e1405be04f26ab28114ef0585e2cac832d3d4

**Comment by pedro-mendonca** [comment]
The commits on this PR were still made partially by rule/sniff, that is the reason of so many small things left behind or introduced. The SQL queries format I didn't know what would be best.
Sorry for having so many details to check.
The next fixes are much better.
Thanks for your help @ocean90

File: gp-includes/things/original.php (hunk 5 above)
Comment (pedro-mendonca) [reply gp-includes/things/original.php]: Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1084/commits/a68e1405be04f26ab28114ef0585e2cac832d3d4

File: gp-includes/things/project.php (hunk 6 above)
Comment (pedro-mendonca) [reply gp-includes/things/project.php]: Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1084/commits/a68e1405be04f26ab28114ef0585e2cac832d3d4

File: gp-includes/things/translation-set.php (hunk 7 above)
Comment (pedro-mendonca) [reply gp-includes/things/translation-set.php]: Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1084/commits/a68e1405be04f26ab28114ef0585e2cac832d3d4

File: gp-includes/things/translation-set.php (hunk 8 above)
Comment (pedro-mendonca) [reply gp-includes/things/translation-set.php]: Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1084/commits/a68e1405be04f26ab28114ef0585e2cac832d3d4

**Review by ocean90 — APPROVED** [review]

**Pull request merged**